mpmp/src/cmds/Subscribe.java

```diff
@@ -25,12 +25,7 @@ public void exec(String line, Conn conn) {
 			return;
 		}
 
-		for (String s : Arrays.copyOfRange(args, 3, args.length)) {
-			if (name == null)
-				name = s;
-			else
-				name = name + " " + s;
-		}
+		name = String.join(" ", Arrays.copyOfRange(args, 3, args.length));
```

Comment:
Yeah, I am just "how can you finish this with basic stuff and fast.
And I somehow managed to implement whispering#2, although you didn't trust me :neutral_face: